aspnetcore/publishing/iis.md

```diff
@@ -34,29 +34,33 @@ Enable the **Web Server (IIS)** server role and establish role services.
 
 Navigate to **Control Panel > Programs > Programs and Features > Turn Windows features on or off** (left side of the screen). Open the group for **Internet Information Services** and **Web Management Tools**. Check the box for **IIS Management Console**. Check the box for **World Wide Web Services**. Accept the default features for **World Wide Web Services** or customize the IIS features to suit your needs.
 
-![image](iis/_static/wf.png)
+![IIS Management Console and World Wide Web Services are selected in Windows Features.](iis/_static/windows-features-win10.png)
 
 ### Windows Server operating systems
 
-For server operating systems, use the **Add Roles and Features Wizard** via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
+For server operating systems, use the **Add Roles and Features** wizard via the **Manage** menu or the link in **Server Manager**. On the **Server Roles** step, check the box for **Web Server (IIS)**.
 
-![image](iis/_static/rs.png)
+![The Web Server IIS role is selected in the Select server roles step.](iis/_static/server-roles-ws2016.png)
 
 On the **Role services** step, select the IIS role services you desire or accept the default role services provided.
 
-![image](iis/_static/role-services.png)
+![The default role services are selected in the Select role services step.](iis/_static/role-services-ws2016.png)
 
-Proceed through the **Confirmation** step to enable the web server role and services.
-
-<a name=unlock-handlers></a>
+Proceed through the **Confirmation** step to install the web server role and services. A server/IIS restart is not required after installing the Web Server (IIS) role. We will execute a server/IIS restart after installing the .NET Core Windows Server Hosting bundle in the next step.
 
 ## Install the .NET Core Windows Server Hosting bundle
 
 1. Install the [.NET Core Windows Server Hosting](https://aka.ms/dotnetcore_windowshosting_1_1_0) bundle on the server. The bundle will install the .NET Core Runtime, .NET Core Library, and the ASP.NET Core Module. The module creates the reverse-proxy between IIS and the Kestrel server.
 
-2. Restart the server or execute **net stop was /y** followed by **net start w3svc** from the command-line to pickup changes to the system PATH.
+2. Restart the server or execute **net stop was /y** followed by **net start w3svc** from a command prompt to pickup changes to the system PATH.
+
+> [!NOTE]
+> If you only plan to host self-contained applications and thus don't require the .NET Core runtime on the server, you have the option of only installing the ASP.NET Core Module by running the installer from an Administrator command prompt: **DotNetCore.1.1.0-WindowsHosting.exe OPT_INSTALL_LTS_REDIST=0 OPT_INSTALL_FTS_REDIST=0**
 
-For more information, see [ASP.NET Core Module](../fundamentals/servers/aspnet-core-module.md).
+> [!NOTE]
```

Comment:
Is this the right location for this note -- is it something you need to know before you do step 1? Or is it something you need to do after step 1 or step 2?